content/_index.fr.html

```diff
@@ -3,69 +3,75 @@
-{{% /blocks/lead %}}
+		{{% blocks/lead color="primary" %}}
+		Une application web, open source, de conception d'infrastructure ferroviaire, d'horaires, d'analyse de capacité
+		et bien plus !
+		{{% /blocks/lead %}}
 
-{{< blocks/section color="dark" type="row justify-content-center" >}}
+		{{< blocks/section color="dark" type="row justify-content-center">}}
 
-{{% blocks/feature icon="fa-solid fa-toolbox" title="Conception ferroviaire" %}}
-Modifiez l’infrastructure ferroviaire et les grilles horaires
-{{% /blocks/feature %}}
+			{{% blocks/feature icon="fa-solid fa-toolbox" title="Conception ferroviaire" %}}
+			Modifiez l’infrastructure ferroviaire et les grilles horaires
+			{{% /blocks/feature %}}
 
-{{% blocks/feature icon="fas fa-microscope" title="Analyse de capacité" %}}
-Détectez les conflits et visualisez la capacité
-{{% /blocks/feature %}}
+			{{% blocks/feature icon="fas fa-microscope" title="Analyse de capacité" %}}
+			Détectez les conflits et visualisez la capacité
+			{{% /blocks/feature %}}
 
-{{% blocks/feature icon="fas fa-drafting-compass" title="Planning court-terme" %}}
-Ajoutez automatiquement de nouveaux trains dans une grille existante
-{{% /blocks/feature %}}
+			{{% blocks/feature icon="fas fa-drafting-compass" title="Planning court-terme" %}}
+			Ajoutez automatiquement de nouveaux trains dans une grille existante
+			{{% /blocks/feature %}}
 
-{{< /blocks/section >}}
+			{{< /blocks/section>}}
 
-{{< blocks/section color="dark" type="row justify-content-center" >}}
+				{{< blocks/section color="dark" type="row justify-content-center">}}
 
-{{% blocks/feature icon="fab fa-github" title="Développement Open source" url="./about/opensource/" %}}
-Tout le monde peut utiliser, redistribuer et développer OSRD
-{{% /blocks/feature %}}
+					{{% blocks/feature icon="fab fa-github" title="Développement Open source" url="./about/opensource/"
+					%}}
+					Tout le monde peut utiliser, redistribuer et développer OSRD
+					{{% /blocks/feature %}}
 
-{{% blocks/feature icon="fas fa-university" title="Gouvernance ouverte" url="./about/governance/" %}}
-OSRD est conçu de manière partagée et publique, les décisions sont prises collégialement
-{{% /blocks/feature %}}
+					{{% blocks/feature icon="fas fa-university" title="Gouvernance ouverte" url="./about/governance/"
+					%}}
+					OSRD est conçu de manière partagée et publique, les décisions sont prises collégialement
+					{{% /blocks/feature %}}
 
-{{% blocks/feature icon="fas fa-exchange-alt" title="Conçu pour l'interopérabilité" %}}
-Faites vos simulations à l’aide de formats d'infrastructure et de systèmes de signalisation propriétaires
-{{% /blocks/feature %}}
+					{{% blocks/feature icon="fas fa-exchange-alt" title="Conçu pour l'interopérabilité" %}}
+					Faites vos simulations à l’aide de formats d'infrastructure et de systèmes de signalisation
+					propriétaires
```

Comment:
nit: To avoid confusion with open/proprietary formats
```suggestion
					sur mesure
```